transitive_deps.bzl

```diff
@@ -6,4 +6,7 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 def cargo_raze_transitive_deps():
     """Loads all dependnecies from repositories required for cargo-raze"""
     rules_foreign_cc_dependencies()
-    rust_repositories(include_rustc_srcs = True)
+    rust_repositories(
+        edition = "2018",
```

Comment:
This change is needed due to https://github.com/bazelbuild/rules_rust/pull/1256.